arch/x86/chunkset_sse41.c

```diff
@@ -0,0 +1,89 @@
+/* chunkset_sse2.c -- SSE2 inline functions to copy small data chunks.
+ * For conditions of distribution and use, see copyright notice in zlib.h
+ */
+
+#include "zbuild.h"
+
+#ifdef X86_SSE41
+#include <immintrin.h>
+#include "chunk_permute_table.h"
+
+typedef __m128i chunk_t;
+
+#define CHUNK_SIZE 16
+
+#define HAVE_CHUNKMEMSET_1
+#define HAVE_CHUNKMEMSET_2
+#define HAVE_CHUNKMEMSET_4
+#define HAVE_CHUNKMEMSET_8
+#define HAVE_CHUNK_MAG
+
+static const lut_rem_pair perm_idx_lut[29] = {
+    {0, 1},      /* 3 */
+    {0, 0}, /* don't care */
```

Comment:
Maybe comments should be aligned.

Reply:
Looks like maybe you may have forgotten this one. Or aligned a different table..